splunk_topology/check.py

```diff
@@ -139,8 +141,10 @@ def _dispatch_and_await_search(self, instance, saved_searches):
                     self._finalize_sid(instance, sid, saved_search)
                     self.update_persistent_status(instance.instance_config.base_url, saved_search.name, sid, 'remove')
             except FinalizeException as e:
-                self.log.error("Got an error %s while finalizing the saved search %s" % (e.message, saved_search.name))
-                raise e
+                if not instance.splunk_ignore_saved_search_errors:
+                    self.log.error("Got an error %s while finalizing the saved search %s" % (e.message, saved_search.name))
+                    raise e
+                self.log.warning("Ignoring finalize exception as ignore_saved_search_errors flag is true.")
```

Comment:
maybe log the actual error when the flag is set to true

Reply:
maybe include a piece of specific information in where the error occurred? This message is very similar as to previous message. This message may be fired on dispatching while the other might be fired during finalization.

Reply:
showing the actual error as well.